plugins/GranularPitchShifter/GranularPitchShifterEffect.cpp

```diff
@@ -157,17 +157,17 @@ bool GranularPitchShifterEffect::processAudioBuffer(SampleFrame* buf, const fpp_
 		if (++m_timeSinceLastGrain >= m_nextWaitRandomization * waitMult)
 		{
 			m_timeSinceLastGrain = 0;
-			double randThing = (fast_rand()/static_cast<double>(FAST_RAND_MAX) * 2. - 1.);
+			double randThing = (fast_rand() * FAST_RAND_RATIO * 2. - 1.);
```

Comment:
@LostRobotMusic I did some random number changes in granular pitch shifter code. It's technically harmless, but would like your heads up as an assurance for me.

Reply:
Your changes should work fine.  I do recommend still using `static_cast<double>` on that first instance though.  Otherwise the compiler will leave it as a float and it'll need to do float-to-double conversions in realtime.

Reply:
Thanks for the pick. Done